stdlib/public/core/Algorithm.swift

```diff
@@ -63,15 +65,15 @@ public func max<T : Comparable>(_ x: T, _ y: T, _ z: T, _ rest: T...) -> T {
 ///     iterator.next() // (1, "bar")
 ///     iterator.next() // nil
 ///
-/// - Note: Idiomatic usage is to call `enumerate` instead of
+/// - Note: Idiomatic usage is to call `enumerated()` instead of
 ///   constructing an `EnumerateIterator` directly.
 public struct EnumeratedIterator<
   Base : IteratorProtocol
 > : IteratorProtocol, Sequence {
   internal var _base: Base
   internal var _count: Int
 
-  /// Construct from a `Base` iterator.
+  /// Initialize from a `Base` iterator.
```

Comment:
The terminology change here is interesting.  We use `construct` fairly liberally around initializers in the rest of the docs.  Why change this specifically?